src/odemis/gui/cont/fastem_project_grid_base.py

```diff
@@ -149,6 +151,67 @@ def ApplyEdit(self, row, col, grid):
         grid.SetCellValue(row, col, new_value)
 
 
+class GridCellFloatRangeEditor(GridCellFloatEditor):
+    """
+    GridCellFloatRangeEditor(min=None, max=None, width=-1, precision=-1, format=GRID_FLOAT_FORMAT_DEFAULT)
```

Comment:
the doc string of class and init is the same. Maybe elaborate it as it is not providing much information esp. the first line in the doc string?